src/side-navigation/side-navigation.scss

```diff
@@ -152,16 +139,7 @@
       }
     }
 
-    &:focus {
-      outline: 0;
-      @include themed {
-        box-shadow: inset 0 0 $iui-xs rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-      }
-    }
-
-    &:focus:not(:focus-visible) {
-      box-shadow: none;
-    }
+    @include iui-focus($offset: -2px, $thickness: 2px);
```

Comment:
Line 142 is actually not needed because it is inherited from `.iui-button .iui-default` so it has been removed.  The only other mention of `:focus` & `:focus-visible` that I'm seeing is the `border-color` change which I think is still necessary.

Reply:
It will not be necessary if we switch our focus mixin to use the `@supports` fallback

Reply:
I tried this but it doesn't work:
```scss
&,
&:hover,
&[disabled],
@include focus-visible {
  @include themed {
    border-color: t(iui-color-background-5);
  }
}
```
I was at least able to simplify to this:
```scss
&,
&:hover,
&[disabled],
&:focus,
&:focus:not(:focus-visible) {
  @include themed {
    border-color: t(iui-color-background-5);
  }
}
```

Reply:
By "not necessary", I meant you could just remove it, as focus rules are not affecting border anymore.
```scss
&,
&:hover,
&[disabled] {
  @include themed {
    border-color: t(iui-color-background-5);
  }
}
```

Reply:
Ahh, I understand now.  Removed `&:focus, &:focus:not(:focus-visible)` from that part of the SCSS and confirmed that everything is displaying properly.